js/main.js

```diff
@@ -27,6 +27,10 @@ function launchListSwitch(prevOrUpcoming) {
     }
     $loadingScreen.className = 'loading-screen hidden';
   });
+  xhrLaunches.addEventListener('error', () => {
+    window.alert('Unable to retreive launch data at this time');
```

Comment:
While `alert`s are ok for errors, in-page messages are a better solution for long-term usability.